src/core/render/index.js

```diff
@@ -444,7 +445,7 @@ export function initRender(vm) {
   }
 
   // Add nav
-  if (config.loadNavbar) {
+  if (config.loadNavbar && !config.nav_el) {
```

Comment:
- [ ] I think this parts needs to be reworked a little.
  - Unless `mergeNavbar` is `true`, `navAppendToTarget` should be ignored if `nav_el` is defined so that `nav_el` remains in its original place.
  - If `mergeNavbar` is true, I think we should either
    - Move the custom navbar into the sidebar so the behavior is as expected, or
    - Just place Docsify's custom nav in the sidebar for mobile
    - Which of the previous two sounds better? Perhaps the second one.

  Someone who wants a completely custom navbar without loading _navbar.md files would just leave `loadSidebar` and `mergeNavbar` undefined. They could additionally enable either of the two and get either behavior or enable both.

Reply:
> If `mergeNavbar` is true, just place Docsify's custom nav in the sidebar for mobile

Should the existing `navEl` be left in the DOM or removed in that case?

Reply:
I guess that's what you meant by the first option.

I don't know what the implications of leaving an empty navEl around would be, so I can't say which one is better.

Reply:
So this is the logic I think we want?

| navEl | mergeNavbar | isMobile | action |
|---|---|---|---|
| 0 | 0 | 0 | append to dom.body |
| 0 | 0 | 1 | append to dom.body |
| 0 | 1 | 0 | append to dom.body |
| 0 | 0 | 1 | append to sidebar |
| 1 | 0 | 0 | no action |
| 1 | 0 | 1 | no action |
| 1 | 1 | 0 | no action |
| 1 | 1 | 1 | append to sidebar |